src/main/java/core/model/player/Player.java

```diff
@@ -2462,4 +2459,42 @@ public long calculateTSI() {
         return round(tsi/10) * 10;
     }
 
+    /**
+     * Cost to convert a player to coach
+     * <p>
+     * Price(sk) = 47.500.000 * a / (ExpLevel - 1)
+     * <p>
+     * Where a = 7.11 for excellent, 1.00 for solid, 0.296 for passable and 0.037 for inadequate coach
+     * ExpLevel = experience level of the player, considering the sublevel, if known
+     * <p>
+     * Notes:
+     * <p>
+     * 1 - For ExpLevel, disastrous is 1.0-2.0, wretched 2.0-3.0, etc...
+     * 2 - For those who have foxtrick, an estimation of the experience sublevel can be found in the players page in the link "Performance history".
+     * 3 - Example: a player with experience level titanic (15.3) would cost 47 500 000 * 7.11 / (15.3 - 1) = € 2.36kk to convert to an excellent coach.
+     * 4 - The cost to convert a player to weak coach is always € 10k.
+     *
+     * @param trainerSkill 4..8
+     * @return HOCurrency
+     */
+    public HOCurrency calculateCoachConversionCosts(int trainerSkill){
+        var experience = this.getSkill(EXPERIENCE);
+        if ( experience >= trainerSkill && trainerSkill >= 4 && trainerSkill < 9){
+            if ( trainerSkill >= 5){
+                var a = switch (trainerSkill) {
+                    case 5 -> 0.037;
+                    case 6 -> 0.296;
+                    case 7 -> 1;
+                    case 8 -> 7.11;
+                    default -> throw new IllegalStateException("Unexpected value: " + trainerSkill);
+                };
+                return new HOCurrency((int)(47500000 * a / (experience-1)));
```

Comment:
In my old age, I find it hard to parse numbers above thousands.  Would you be able to write `47_500_000` for clarity?  Same for `100_000` below.

Reply:
Thanks - again i learned new java stuff.